internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -94,6 +95,8 @@ public class VeniceServerWrapper extends ProcessWrapper implements MetricsAware
   public static final String SERVER_ENABLE_SSL = "server_enable_ssl";
   public static final String SERVER_SSL_TO_KAFKA = "server_ssl_to_kafka";
   public static final String CLIENT_CONFIG_FOR_CONSUMER = "client_config_for_consumer";
+  public static final String SERVER_DELETE_UNASSIGNED_PARTITIONS_ON_START =
```

Comment:
Let's reuse the same config (SERVER_DELETE_UNASSIGNED_PARTITIONS_ON_STARTUP) name instead of creating new feature flag for test only usage

Reply:
Made the mistake of opening up a PR against a branch in linkedin/venice.

Created one against my personal one.

https://github.com/linkedin/venice/pull/1383